### PR TITLE
feat: update wallets page

### PR DESCRIPTION
**Pull request by secondl1ght**
As talked about in issue #59, I have made some updates to the wallets page. I added an Active indicator and also moved the buttons inline. This matches better with the Figma design and I think looks a bit better.

# Before:
image

# After:
image

### CODE REVIEW

**Review by dnlggr — COMMENTED** [review]
Cool, thanks for taking that on! 👌 

Looking at Figma, I feel like we can still improve the looks a bit. Let's go with a smaller heading, use the same type of buttons as on the main wallet page and go for the lighter look with the dividers instead of the wallet cards.

For centering the whole thing I went with the following configuration for the "create wallet" page since we don't have a global layout container yet. Maybe you can reuse it here as well. Eventually we should do DRY this up and not repeat it on every page, but that's definitely out of scope for this PR.

``` js
<rb.Row className="justify-content-center">
  <rb.Col md={10} lg={8} xl={6}>
    // ...
  </rb.Col>
</rb.Row>
```

Let me know if you need anything!

Screenshot 2022-02-08 at 10 26 20

**Comment by dergigi** [comment]
Great stuff @secondl1ght! I agree with Daniel, let's see if we can get a bit closer to the design and make the wallets page shine! 😊 ✨

**Comment by secondl1ght** [comment]
Ok I made more changes, here's how it looks now:

image

image

File: src/components/Wallet.jsx [hunk 1]
```diff
@@ -113,75 +113,90 @@ export default function Wallet({ name, currentWallet, startWallet, stopWallet, s
...
+                  <rb.Card.Title className={!isActive && !noneActive && 'mb-0'}>
+                    {walletDisplayName(name)}
+                  </rb.Card.Title>
+                  {isActive ? <span className="text-success">Active</span> : <></>}
+                </div>
+                <div>
+                  {isActive ? (
+                    hasToken ? (
+                      <>
+                        <Link className="btn btn-outline-dark me-2" to="/wallet">
```
Comment (dnlggr) [inline src/components/Wallet.jsx:132]: Same comment as above for the remaining buttons here.

**Review by dnlggr — COMMENTED** [review]
Cool thanks for the update—it looks great already!

I left a few suggestion inline. It's not much but could be worth adding for some extra sparkels. ✨ 😄  Let me know what you think!

File: src/components/Wallet.jsx [hunk 2]
```diff
@@ -113,75 +113,90 @@ export default function Wallet({ name, currentWallet, startWallet, stopWallet, s
...
+                          Display
```
[the remark this answers is not in the document]
Comment (theborakompanioni) [reply src/components/Wallet.jsx:133]: > I feel like "Open" would sound more natural than "Display". What do you think?
 
waxwing mentioned that in the chat (it was labeled "Open" previously):
 > i have one more nit: when the wallet is unlocked, the main page shows 'open', maybe it should say 'display'?

This is just a hint at why it is labeled like that - not a vote to keep it that way.

File: src/components/Wallets.jsx [hunk 3]
```diff
@@ -69,10 +69,11 @@ export default function Wallets({ currentWallet, startWallet, stopWallet }) {
           setAlert={setAlert}
         />
       ))}
-
-      <Link to="/create-wallet" className={`btn mt-3 ${walletList?.length === 0 ? 'btn-dark' : 'btn-outline-dark'}`}>
-        Create Wallet
-      </Link>
+      <div className="d-flex justify-content-center">
+        <Link to="/create-wallet" className={`btn mt-3 ${walletList?.length === 0 ? 'btn-dark' : 'btn-outline-dark'}`}>
```
[the remark this answers is not in the document]
Comment (secondl1ght) [reply src/components/Wallets.jsx:73]: Which button are you referring to? This one?

image

File: src/components/Wallet.jsx [hunk 4]
```diff
@@ -113,75 +113,90 @@ export default function Wallet({ name, currentWallet, startWallet, stopWallet, s
...
+      <rb.Col md={10} lg={8} xl={6}>
+        <rb.Card className="mt-3 bg-transparent border-start-0 border-end-0 rounded-0" {...props}>
+          <rb.Card.Body>
+            <rb.Form onSubmit={onSubmit} validated={validated} noValidate>
+              <div className="d-flex justify-content-between align-items-center flex-wrap">
+                <div>
+                  <rb.Card.Title className={!isActive && !noneActive && 'mb-0'}>
+                    {walletDisplayName(name)}
+                  </rb.Card.Title>
+                  {isActive ? <span className="text-success">Active</span> : <></>}
```
[the remark this answers is not in the document]
Comment (secondl1ght) [reply src/components/Wallet.jsx:122]: Good idea, I think this looks really good!

File: src/components/Wallets.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (dnlggr) [reply src/components/Wallets.jsx:73]: Yeah this one and the withdraw and deposit buttons. Specifically the flatter look with a light border and a more bold text. That way we'll get closer to a seamless and consistent UI across different screens.

**Comment by secondl1ght** [comment]
Extra sparkles are always good! Here is the latest screenshot, let me know if this is what you had in mind:

image

**Review by dnlggr — APPROVED** [review]
I pushed a small update with some smaller fixes for the buttons for the light theme and a small alignment change. Hope that is alright with you. Thanks again for the contribution! Let's get this in! 🚀

**Pull request merged**